package.json

```diff
@@ -5,7 +5,7 @@
   "main": "app.js",
   "scripts": {
     "start": "node .",
-    "test": "NODE_ENV=test mocha",
+    "test": "NODE_ENV=test mocha test/*",
```

Comment:
DANIELE where you have been, haven't had you review my PRs in aaages.  mocha --recursive worked like a charm

So I was taught that my test folder structure should follow the same structure as my app, that's why I have a lib folder.  The idea was so that it's easy to find the related test.  Is that not necessarily true? Have you got a different thought?